### ISSUE TITLE
Add multiple blockable addresses

### ISSUE THREAD

**Issue by alve89** [alve89/j_plg_hrz_disablelogin#9]
From #8

P.S.: Just an idea to possible/future plugin: after to use, install and test this plugin I realized that could be useful too if there is a similar plugin to restrict access to one or more URLs. Plugin would work very similar and block access to https://mydomain.tld/path but allow access if provide a correct secret key: https://mydomain.tld/path&YOUR_SECRET_KEY.
On plugin configuration form user should fill and set:

URL (text input)
Secret Key (text input)
Add more (button) <-- to add a new URL and Secret Key
Enable Logging (boolean)
Show access denial in frontend (boolean)
Target-address on blocked access (text input)

Probably combination of `plg_hrz_block_access` and `plg_hrz_disablelogin` could be the best solution.

**Comment by filipe-torres** [comment]
Yes, a combination of `plg_hrz_block_access` and `plg_hrz_disablelogin` plugins plus this new feature is a great idea and the best solution, but I think you should create a new plugin project merging these functionalities all together, since `plg_hrz_block_access` main purpose is just block unwanted access to login form.
